customer/src/test/java/no/unit/nva/customer/get/ListAllChannelClaimsHandlerTest.java

```diff
@@ -85,14 +89,32 @@ void shouldReturnOkWhenListingChannelClaims() throws IOException, ApiGatewayExce
         assertThat(channelClaimsListResponse.channelClaims().size(), is(4));
     }
 
-    private void insertRandomCustomerWithChannelClaim(List<ChannelClaimDto> channelClaims) throws ApiGatewayException {
+    @Test
+    void shouldReturnOkAndListChannelClaimsForInstitutionProvidedInQueryParam() throws IOException,
+                                                                                       ApiGatewayException {
+        var customer = insertRandomCustomerWithChannelClaim(List.of(randomChannelClaimDto(), randomChannelClaimDto()));
```

Comment:
I believe there exists a method that return a list of three random channel claims.

```suggestion
        var customer = insertRandomCustomerWithChannelClaim(randomChannelClaimDtos());
```